pkg/controller/bootstrap/tokencleaner.go

```diff
@@ -52,6 +54,7 @@ type TokenCleanerOptions struct {
 func DefaultTokenCleanerOptions() TokenCleanerOptions {
 	return TokenCleanerOptions{
 		TokenSecretNamespace: api.NamespaceSystem,
+		SecretResync:         defaultSecretResyncInterval,
```

Comment:
this is an **extremely** short resync period. I probably wouldn't set this below an hour

Reply:
I looked at the kubeadm command and did not provide a valid default value. The token controller is only used for gc. It is reasonable to set a longer time. I have modified it to an hour

Reply:
what is the offset from the expected GC time?
are you seeing a resync that never happens?

Reply:
I understand that the token controller will only enqueue the object when adding or modifying the secret. Add a resync in order to re-enqueue all the secrets.

> what is the offset from the expected GC time?
Currently set to one hour, maybe longer？